### DISCUSSION TITLE
Validator's info discussion

### DISCUSSION THREAD

**Discussion by bro-n-bro-0** [bro-n-bro/bro.app#88] [Ideas]
## 👋 Welcome!
 
Here we can discuss the validator info page. Do not hesitate to add some params and criteria you want to know about the validator

**Comment by bro-n-bro-2** [comment]
Slashing count
commission earned
double sign (?)
uptime (?)
unique delegators (?)
stake value in USD (?)
Restake enabled (?)

**Comment by bro-n-bro-99** [comment]
- [ ] voting participation - the percentage of voting proposals in which the validator has participated
- [ ] proposals sponsored - the number of governance proposals supported by the validator (rejected / abstained)
- [ ] historical rewards - the cumulative rewards earned by the validator over time (graph)
- [ ] reputation score -> reference to the bro score?
- [ ] delegation changes over time - the history of stake changes for the validator, including additions and withdrawals
- [ ] total stake delegated to the validator - the sum of tokens delegated to the validator by all delegators
- [ ] average return on delegation - the average percentage return earned by delegators who have staked with the validator
- [ ] historical fee changes - the graph showing historical commision changes over time
- [ ] software upgrades - is it possible to track the time that took a validator to do the upgrade?

**Comment by bro-n-bro-99** [comment]
- [ ] gas usage efficiency - the efficiency of the validator in utilizing gas during block validation (?)
- [ ] new delegators - the number of new delegators who have recently started staking with the validator (last 30 days data?)

**Comment by freak12techno** [comment]
on voting participation: it's important to keep track whether the validator was in the active set/created when the proposal took place, to not respect these proposals either a validator could not vote upon because it was not created, or it was inactive so its vote would not count
